src/plugins/sailfishoswizards/data/qml-modules.json

```diff
@@ -0,0 +1,22 @@
+{
+    "QtPositioning 5.3": {
+        "layoutname": "Qt Positioning",
+        "qt": ["positioning"],
+        "yamlheader": "Requires:",
+        "specheader": "Requires",
+        "dependencies": ["sailfishsilica-qt5 >= 0.10.9", "qt5-qtdeclarative-import-positioning"],
+        "types": ["PositionSource"]
+	},
+    "QtSensors 5.11": {
+        "layoutname": "Qt Sensors",
+        "qt": ["sensors"],
+        "yamlheader": "Requires",
+        "specheader": "Requires",
+        "dependencies": ["sailfishsilica-qt5 >= 0.10.9", "qt5-qtdeclarative-import-sensors"],
+        "types": ["Accelerometer", "AccelerometerReading", "Altimeter", "AltimeterReading", "AmbientLightReading", "AmbientLightSensor", "AmbientTemperatureReading",
+        "AmbientTemperatureSensor", "Compass", "CompassReading", "DistanceReading", "DistanceSensor", "Gyroscope", "GyroscopeReading", "HolsterReading",
+        "HolsterSensor", "HumidityReading", "HumiditySensor", "IRProximityReading", "IRProximitySensor", "LidReading", "LidSensor", "LightReading", "LightSensor",
+        "Magnetometer", "MagnetometerReading", "OrientationReading", "OrientationSensor", "PressureReading", "PressureSensor", "ProximityReading", "ProximitySensor",
+        "RotationReading", "RotationSensor", "Sensor", "SensorGesture", "SensorGlobal", "SensorReading", "TapReading", "TapSensor", "TiltReading", "TiltSensor"]
+    }
+}
```

Comment:
Available modules, their dependencies and types they provide depend on target OS version. We should allow to choose the target OS version and adjust the offered modules data based on that. Preferably with per-target configuration.

Reply:
Do you mean Sailfish OS version? But what a problem with different Sailfish OS versions?

Reply:
The list of supported modules is not stable - some new are added, some old are dropped as times go. So every Sailfish OS version (and their variants) may be different in this respect.

Reply:
So, where we can take an information about modules and Sailfish OS versions where they are available? And how to update this information when Sailfish OS will be updated?

Reply:
At this level it is not necessary to deal with that question. Just a mechanism needs to be implemented that will enable adding that information.

So e.g. instead of using single hardcoded `qml-modules.json` we could load all json files found under a well known directory (likely a subdirectory of the wizard directory)  and extend the json file format so that the  information about the OS version can be recorded there. Then, in the page where the modules are selected, there would be one more combox box to choose the desired OS version.

Later we would think how to collect the information about available modules for each supported OS version and supply that.

Reply:
In order to be both backwards and forwards compatible, we actually need two mechanisms: 1) For old OS versions, we need the functionality that martyone described above. To be futureproof, we should also have another mechanism, which fetches the information directly from the build target. But I think you shouldn't even try to implement it here and now.

Reply:
Yes, by "Later we would think how to collect the information about available modules for each supported OS version and supply that." I meant that another mechanism.

Reply:
Ok. We undestood about setting the Sailfish OS version and display available QML-types in depend on the version. But what versions should we display for user right now? And where we can find information about types differences between OS versions?

Reply:
In scope of this PR it is not expected to supply any data about available modules. For testing you can use any fake data. Real data with be supplied by the "another mechanism". (The wizard should fail gracefully if no data is supplied)